### ISSUE TITLE
Will it still be actively maintained?

### ISSUE THREAD

**Issue by Platway** [tmpim/casket#25]
I've still been using Caddy v1 as I don't like v2. I found this repo yesterday and decided to upgrade my Caddy v1 to Casket. But I saw the latest release was more than a year ago and would like to know if there's any plan. Or is there any other active Caddy v1 forks out there? Thanks for your great work!

**Comment by Lustyn** [comment]
While this repo is not super active, we are very much still making wide use of Casket (primarily within tmpim). The most notable usage is probably powering [TETR.IO](https://tetr.io/) and [SwitchCraft](https://sc3.io/). Pull requests are welcome for any bug fixes or feature-creep you may encounter along the way.

That being said, there are a few big fish to fry for true maintainability going forward:
1. [Certmagic is sorely out of date](https://github.com/tmpim/casket/issues/19), we are at risk of using deprecation path ACME depending on what direction Let's Encrypt may go in the future. tmpim is mostly missing anyone with a motivation to tackle this, and I probably wouldn't recommend casket for production reliability for this reason. It's still a great tool for personal use or scenarios where you're comfortable exploring a different web server in the future if the project gets too stale.
2. Any plugin ecosystem is missing, and you will lose any benefits of the public builds if you have use-cases that fall outside of ours without making an effort to merge those plugins upstream in [casket-plugins](https://github.com/tmpim/casket-plugins).

Our concerns with forking Caddy were mostly around keeping us within the bounds of what was provided to us by Caddy v1, without having to teach a whole lot of people the new semantics of Caddy v2's configuration format and pulling in a load of unnecessary "cloud-native" features. There are a few big things that pushed us away:
1. Caddy v2 seems focused, as mentioned, on being a "cloud-native" web server a-la Traefik and co. This is great for those in that space, but for a bunch of developers best described as "hackers" who run our own small infrastructure it is biting off far too much to chew.
2. Somewhat a side effect of the above, but Caddy v2 has terrible defaults for the average user. Things like JSON logs, Admin API, are all on by default with difficult at best ways to opt-out of those features.
3. The configuration language was completely replaced, providing completely different semantics more similar to NGINX than what made Caddy good before. It still has automatic certificate management, but that's about all of the previous magic that remains.


To summarize, you're welcome to join us in using this fork, and we would much appreciate any contributions anyone would want to make. But we can't say for certain if that will continue for good 🤷
